src/components/errors/ErrorComms.tsx

```diff
@@ -1,27 +1,24 @@
 import React from 'react'
 import './errors.css'
-import errorImage from '../../images/errors/error-robotdead.png'
+import errorImage from '../../images/errors/error-robotdown.png'
```

Comment:
Is `error-robotdead.png` being used somewhere else? if not, we can remove it.

Reply:
Just removed it!